marklogic-data-hub/src/main/java/com/marklogic/hub/impl/EntityManagerImpl.java

```diff
@@ -244,22 +244,22 @@ private HubModuleManager getPropsMgr() {
         return new HubModuleManager(timestampFile);
     }
 
-    private List<JsonNode> getAllEntities() {
+    public List<JsonNode> getAllEntities() {
         List<JsonNode> entities = new ArrayList<>(getAllLegacyEntities());
```

Comment:
I think this is another reason not to use getAllEntities() - I don't think we want legacy entities included. According to our docs - https://docs.marklogic.com/datahub/upgrade.html - hubUpdate will move legacy entities to the ./entities directory. But in case there's anything left in the legacy entities directory, we don't want to touch those files.